resources/translations/fi.edn

```diff
@@ -454,7 +455,15 @@
            :phase-revoked "Vaihe peruttu:"
            :rejected "Hylätty"
            :revoked "Peruttu"}
-  :profile {:idp-email "Sähköpostiosoite tunnistautumispalvelulta"
+  :profile {:delete-api-key "Poista API-avain"
+            :ega-api-key-expiration-date "API-avain vanhenee"
+            :ega-api-key-expired [:p "Sinulla on vanhentunut API-avain. Luo itsellesi uusi avain."]
+            :ega-api-key-none [:p "Näyttää että sinulla ei ole vielä API-avainta."]
+            :ega-api-key-valid [:p "Sinulla on voimassa oleva API-avain."]
+            :ega-full "The European Genome-phenome Archive"
+            :ega-intro [:p "Hyväksymiesi hakemusten käyttövaltuudet voidaan lähettää EGA-palvelimille. Tämä vaatii että sinulla on voimassa oleva API-avain. Varmista että olet ensin yhdistänyt ELIXIR-käytäjätunnuksesi EGA-tunnukseesi."]
```

Comment:
"Tämä vaatii,"  "Varmista," (<- missing commas)
Also a typo in "ELIXIR-käyt**t**äjätunnuksesi.